portal/templates/flask_user/_macros.html

```diff
@@ -106,7 +106,7 @@
 </div>
 {%- endmacro %}
 {% macro linksHTML(user)-%}
-    <p><small><a href="{{PORTAL}}/about">{{ _("About") }}</a> | <a href="{{PORTAL}}/contact">{{ _("Contact") }}</a>{%- if user -%}&nbsp;| <a href="{{PORTAL}}/privacy">{{ _("Privacy") }}</a>{%- endif -%}{% if user or config.SHOW_PUBLIC_TERMS %}&nbsp;| <a href="{{PORTAL}}/terms">{{ _("Terms") }}</a></small></p>{%- endif -%}
+    <p><small class="links"><a href="{{PORTAL}}/about">{{_("About")}}</a> | <a href="{{PORTAL}}/contact">{{_("Contact")}}</a>{%- if user -%}&nbsp;| <a href="{{PORTAL}}/privacy">{{_("Privacy")}}</a>{%- endif -%}{% if user or config.SHOW_PUBLIC_TERMS %}&nbsp;| <a href="{{PORTAL}}/terms">{{_("Terms")}}</a></small></p>{%- endif -%}
```

Comment:
Can you use `url_for()` for some of these? (can't remember...)

Reply:
@ivan-c ivan, yea, I was running into issue with these endpoints existing in both eproms and gil blueprints respectively - therefore I couldn't use url_for for these.